project/ScalacOptions.scala

```diff
@@ -16,78 +16,85 @@
-    "-feature", // Emit warning and location for usages of features that should be imported explicitly.
-    "-unchecked", // Enable additional warnings where generated code depends on assumptions.
-    "-encoding",
-    "utf-8", // Specify character encoding used by source files.
-    "-explaintypes", // Explain type errors in more detail.
-    // "-language:existentials", // Existential types (besides wildcard types) can be written and inferred
-    "-language:experimental.macros", // Allow macro definition (besides implementation and application)
-    "-language:higherKinds", // Allow higher-kinded types
-    "-language:implicitConversions", // Allow definition of implicit functions called views
-    "-Xcheckinit", // Wrap field accessors to throw an exception on uninitialized access.
-    // "-Xfatal-warnings", // Fail the compilation if there are any warnings.
-    // "-Xfuture", // Turn on future language features.
-    "-Xlint:adapted-args", // Warn if an argument list is modified to match the receiver.
-    // "-Xlint:delayedinit-select", // Selecting member of DelayedInit.
-    "-Xlint:doc-detached", // A Scaladoc comment appears to be detached from its element.
-    "-Xlint:inaccessible", // Warn about inaccessible types in method signatures.
-    "-Xlint:infer-any", // Warn when a type argument is inferred to be `Any`.
-    "-Xlint:missing-interpolator", // A string literal appears to be missing an interpolator id.
-    "-Xlint:nullary-override", // Warn when non-nullary `def f()' overrides nullary `def f'.
-    "-Xlint:nullary-unit", // Warn when nullary methods return Unit.
-    "-Xlint:option-implicit", // Option.apply used implicit view.
-    // "-Xlint:package-object-classes", // Class or object defined in package object.
-    "-Xlint:poly-implicit-overload", // Parameterized overloaded implicit methods are not visible as view bounds.
-    "-Xlint:private-shadow", // A private field (or class parameter) shadows a superclass field.
-    "-Xlint:stars-align", // Pattern sequence wildcard must align with sequence component.
-    "-Xlint:type-parameter-shadow", // A local type parameter shadows a type already in scope.
-    // "-Ywarn-dead-code", // Warn when dead code is identified.
-    // "-Ywarn-numeric-widen", // Warn when numerics are widened.
-    // "-Ywarn-unused:implicits", // Warn if an implicit parameter is unused.
-    // "-Ywarn-unused:locals", // Warn if a local definition is unused.
-    // "-Ywarn-unused:params", // Warn if a value parameter is unused.
-    // "-Ywarn-unused:patvars", // Warn if a variable bound in a pattern is unused.
-    // "-Ywarn-unused:privates", // Warn if a private member is unused.
-    "-Ywarn-value-discard", // Warn when non-Unit expression results are unused.
-    "-Xmacro-settings:show-coder-fallback=true",
-    "-Ydelambdafy:inline", // Set the strategy used for translating lambdas into JVM code to "inline"
-    "-Ybackend-parallelism",
```

Comment:
On further inspection I noticed that bloop and intellij don't really like this setting when `-release 8` is added.

Reply:
let's see how's the compile time without it ...